### DISCUSSION TITLE
How do I ignore curly braces?

### DISCUSSION THREAD

**Discussion by ngaisteve1** [11ty/eleventy#2559] [Uncategorized]
In my html, I have some variables which I wrap with double curly braces. For example {{ name }}. But this is for my own custom program usage. Not for eleventy to evaluate. 

My expected output is  `<p>{{ name }}</p>` but actual output is `<p></p>`.

**Comment by pdehaan** [comment]
You could either wrap the `name` variable in `{% raw %}` tags, like `{% raw %}{{ name }}{% endraw%}` if you still want to use liquid/nunjucks tags in your template.
Or try setting the `htmlTemplateEngine` to `false` in your .eleventy.js config file if you want to disable templating completely for .html files, per https://www.11ty.dev/docs/config/#default-template-engine-for-html-files.

**Reply by pdehaan** [reply]
Actually, another option is to set [`templateEngineOverride`](https://www.11ty.dev/docs/languages/#templateengineoverride-examples) if you only want to disable templating for a specific file:

```njk
---
title: This is my title
templateEngineOverride: false
---

<h1>{{ title }}</h1>

<p>name="{{ name }}"</p>

<pre>{% raw %}{{ title }}{% endraw %}</pre>
```

Which will render the following, since there won't be any liquidjs or nunjucks preprocessing:
```html
<h1>{{ title }}</h1>

<p>name="{{ name }}"</p>

<pre>{% raw %}{{ title }}{% endraw %}</pre>
```
